GameBoard/Areas/Identity/Pages/Account/ResetPassword.cshtml.cs

```diff
@@ -65,7 +72,7 @@ public async Task<IActionResult> OnPostAsync()
 
         public class InputModel
         {
-            [Required]
+            //[Required]
```

Comment:
Required? Not required? Why the comment?

Reply:
Done